bundles/org.openhab.binding.haassohnpelletstove/src/main/resources/OH-INF/i18n/haassohnpelletstove_xx_XX.properties

```diff
@@ -0,0 +1,29 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
You need this file if you want to translate the binding to another language than English. As you have only English, this file could be removed.

Reply:
ok